renderer/brave_content_renderer_client.cc

```diff
@@ -17,3 +20,13 @@ SetRuntimeFeaturesDefaultsBeforeBlinkInitialization() {
   blink::WebRuntimeFeatures::EnableSharedArrayBuffer(false);
 }
 BraveContentRendererClient::~BraveContentRendererClient() = default;
+
+void BraveContentRendererClient::RenderFrameCreated(
+    content::RenderFrame* render_frame) {
+  ChromeContentRendererClient::RenderFrameCreated(render_frame);
+
+  if (render_frame->IsMainFrame()) {
```

Comment:
Just want to double-check that cosmetic filtering will run in subframes as well, I didn't see anywhere that'd be specifically handled

Reply:
as it was discussed in DM, you meant scriplets should run on all frames, css rules only apply to a main frame. So yes, scriplets are done for subframes as well in that place https://github.com/brave/brave-core/pull/6735/files#diff-cfde9d579db91c98f9da4fe25bdea104c9d925ac00346fe153e9d8efa9fffc97R197